x/fungible/keeper/evm.go

```diff
@@ -680,7 +680,7 @@ func (k Keeper) CallEVM(
 // value is the amount of wei to send; gaslimit is the custom gas limit, if nil EstimateGas is used
 // to bisect the correct gas limit (this may sometimes result in insufficient gas limit; not sure why)
 //
-// noEthereumTxEvent flag is used to control if ethereum_tx eventsshould emitted
+// noEthereumTxEvent flag is used to control if ethereum_tx events should be emitted
```

Comment:
As mentioned in the other PR, this is the bug that introduced this option: https://github.com/zeta-chain/node/issues/1010

I'm wondering if the recent changes could have solved the original issue?